aladdin/commands/restart.py

```diff
@@ -12,19 +12,13 @@ def parse_args(sub_parser):
         dest="charts",
         help="Start only these charts (may be specified multiple times)",
     )
-    subparser.add_argument(
```

Comment:
deprecating --with-mount functionality. will recommend people use okteto instead

Reply:
Will that documentation be included in this PR at some point?

Reply:
I'm planning on adding plugins for okteto support with some internal documentation

Reply:
do you think there's no longer value in supporting mounts given okteto? 
and if so then maybe it'd be worth adding the okteto plugins into core aladdin

Reply:
Yeah I wasn't sure which way to go on this. I ended up deciding on aladdin not being okteto aware (i think we felt similarly about vault), and the okteto plugin is currently fivestars specific